src/libs/lib.c

```diff
@@ -990,7 +990,11 @@ static void popup_callback(GtkButton *button, dt_lib_module_t *module)
   mi->version = module->version();
   mi->module = module;
   mi->params = module->get_params ? module->get_params(module, &mi->params_size) : NULL;
+<<<<<<< HEAD
```

Comment:
I have cleaned this before merging, that's why I merged manually.

Reply:
Thanks !